### PR TITLE
Add Java code samples for image redaction

### PR DESCRIPTION
**Pull request by jakubrauch**
To be linked from https://cloud.google.com/dlp/docs/redacting-sensitive-data-images

Fixes #issue

> It's a good idea to open an issue first for discussion.

- [yes] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md)
- [yes] `pom.xml` parent set to latest `shared-configuration`
- [nothing new] Appropriate changes to README are included in PR
- [nothing new] API's need to be enabled to test (tell us)
- [nothing new] Environment Variables need to be set (ask us to set them)
- [yes] Tests pass (`mvn -P lint clean verify`)
  * (Note- `Checkstyle` passing is required; `Spotbugs`, `ErrorProne`, `PMD`, etc. `ERROR`'s are advisory only)
- [yes] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: dlp/src/main/java/dlp/snippets/RedactImageFileAllInfoTypes.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 Google LLC
```
Comment (lesv) [inline dlp/src/main/java/dlp/snippets/RedactImageFileAllInfoTypes.java:2]: 2020 ?

File: dlp/src/main/java/dlp/snippets/RedactImageFileAllText.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2019 Google LLC
```
Comment (lesv) [inline dlp/src/main/java/dlp/snippets/RedactImageFileAllText.java:2]: 2020

File: dlp/src/main/java/dlp/snippets/RedactImageFileColoredInfoTypes.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2019 Google LLC
```
Comment (lesv) [inline dlp/src/main/java/dlp/snippets/RedactImageFileColoredInfoTypes.java:2]: 2020

File: dlp/src/main/java/dlp/snippets/RedactImageFileAllInfoTypes.java (hunk 1 above)
Comment (jakubrauch) [reply dlp/src/main/java/dlp/snippets/RedactImageFileAllInfoTypes.java]: Done

File: dlp/src/main/java/dlp/snippets/RedactImageFileAllText.java (hunk 2 above)
Comment (jakubrauch) [reply dlp/src/main/java/dlp/snippets/RedactImageFileAllText.java]: Done

File: dlp/src/main/java/dlp/snippets/RedactImageFileColoredInfoTypes.java (hunk 3 above)
Comment (jakubrauch) [reply dlp/src/main/java/dlp/snippets/RedactImageFileColoredInfoTypes.java]: Done

**Pull request merged**